data-es7.js

```diff
@@ -738,7 +763,10 @@ exports.tests = [
   category: 'strawman',
   link : 'https://github.com/DavidBruant/Map-Set.prototype.toJSON',
   exec: function(){/*
-    return JSON.stringify(new Map([['a', 'b'], ['c', 'd']])) === '[["a","b"],["c","d"]]';
+    var map = new Map();
+    map.set('a', 'b');
+    map.set('c', 'd');
+    return JSON.stringify(map) === '[["a","b"],["c","d"]]';
```

Comment:
My personal policy for ES6 extensions in ES7 is this: some platforms are implementing ES7 features alongside ES6 - consider that Chrome supports `Object.observe` but not destructuring. Therefore, assumptions about the native ES6 implementation must not be made. Continue to write tests using mainly ES3, with only the minimum of ES5 and ES6 features deployed where strictly necessary, e.g. `JSON.stringify` for a `toJSON` test, or very basic class syntax for the class properties tests.